<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPage</class>
 <widget class="QWidget" name="RegisterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>586</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RegisterPage{
	background-color: rgb(0, 8, 22);
}
#lw_icon{
	background-color:transparent;
	color:rgb(255,255,255);
}
#pic_cover{
	border-image: url(:/image/preview.png);
}

#bar_cover{
color:rgb(28,235,235);
font-size:25px;
padding-bottom:7px;
border-image:url(:/image/title.png);
}


#pb_add{
color:rgb(28,235,235);
font-size:20px;
border-image:url(:/image/add.png);
}
#pb_ensure{
color:rgb(28,235,235);
font-size:20px;
border-image:url(:/image/ensurebutton.png);
}
#pb_cancel{
color:rgb(28,235,235);
font-size:20px;
border-image:url(:/image/ensurebutton.png);
}


.QLineEdit{
border-image:url(:/image/input.png);
color:rgb(16,115,115);
font-size:18px;
}</string>
  </property>
  <widget class="QListWidget" name="lw_icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>480</width>
     <height>183</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QWidget" name="pic_cover" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>192</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_picture">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>192</width>
     <height>320</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_staffno">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入工号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_add">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>347</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bar_cover">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>481</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>人脸注册</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_ensure">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>80</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>80</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <zorder>lb_picture</zorder>
  <zorder>lw_icon</zorder>
  <zorder>pic_cover</zorder>
  <zorder>le_staffno</zorder>
  <zorder>le_name</zorder>
  <zorder>pb_add</zorder>
  <zorder>bar_cover</zorder>
  <zorder>pb_ensure</zorder>
  <zorder>pb_cancel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
